contributors/devel/developer-guides/vagrant.md

```diff
@@ -227,7 +227,7 @@ my-nginx   3         3         3            3           1m
 
 We did not start any Services, hence there are none listed. But we see three
 replicas displayed properly. Check the
-[guestbook](https://github.com/kubernetes/kubernetes/tree/%7B%7Bpage.githubbranch%7D%7D/examples/guestbook)
+[guestbook](https://github.com/kubernetes/examples/tree/master/guestbook)
```

Comment:
Please actually use this URL: https://git.k8s.io/examples/guestbook

Reply:
sorry, didn't notice that, fixed.

Reply:
Still doesn't appear to be updated

Reply:
Yeah,,, I changed it to the url that https://git.k8s.io/examples/guestbook jump to, or maybe I should changed the url to this one https://kubernetes.io/docs/tutorials/stateless-application/guestbook/ ?